### PR TITLE
Front end init

### PR DESCRIPTION
**Pull request by wong-alan**
It could be improved, but it seems mostly functional right now. 
Access the page at /boids/boids.html, and there should be a results.js in the boids folder for the dogs to work.

### CODE REVIEW

File: src/util.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+module.exports = {serveFile}
```
Comment (thethomasz) [inline src/util.js]: is this file even used?

Comment (wong-alan) [reply src/util.js]: Hmm.. I don't think so. Good catch. I think I started writing something and then abandoned it.

**Comment by thethomasz** [comment]
you merged the branch with itself? im confused by that

**Comment by wong-alan** [comment]
> you merged the branch with itself? im confused by that

My local branch was behind I think. So I had to pull and merge.

**Pull request merged**